apps/complaints/index.js

```diff
@@ -1,16 +1,27 @@
 'use strict';
 
+const controllers = require('hof').controllers;
+
 module.exports = {
 	name: 'complaints',
 	steps: {
-		'/': {
-			template: 'step',
-			fields: ['name-text'],
-			next: '/step1'
-		},
-		'/step1': {
-			template: 'step',
-			fields: ['name-text']
+    '/': {
+      controller: controllers.start,
+      next: '/who'
+    },
+		'/who': {
+			fields: ['who'],
+			next: '/applicant-name',
+      forks: [{
+        target: '/declaration',
+        condition: {
+          field: 'who-radio',
+          value: 'representative'
+        }
+      }],
+      locals: {
+        section: 'personal-contact-details'
+      }
 		}
 	}
 };
```

Comment:
Could you check the indentation of this file? It looks like the lines you have amended are spaces and the ones you haven't (6, 7, 25, 26) are tabs